task/import.go

```diff
@@ -75,10 +76,13 @@ func TaskImport(tctx *TaskContext) (*data.TaskOutput, error) {
 		return nil, err
 	}
 	defer importedFile.Close()
-	// RecordStream on output file for HLS playback
-	playbackRecordingId, err := Prepare(tctx, metadata.AssetSpec, importedFile)
-	if err != nil {
-		glog.Errorf("error preparing imported file assetId=%s err=%q", tctx.OutputAsset.ID, err)
+	playbackRecordingId := ""
```

Comment:
Might also add a comment here just in case that this is a temporary hack.
```suggestion
	playbackRecordingId := ""
	// Temporarily skip preparing recorded streams while we figure out a bug in the orchestrators latest version.
	// TODO: Remove this check and prepare all assets.
```